server/models/users.py

```diff
@@ -38,14 +51,42 @@ def delete_all(cls):
         except:
             return {'message': 'Something went wrong'}
 
-    @staticmethod
-    def generate_hash(password):
-        return sha256.hash(password)
 
     @staticmethod
-    def verify_hash(password, hash):
-        return sha256.verify(password, hash)
+    def encode_auth_token(user_id: int):
+        """
+        Generates the Auth Token
+        :return: string
+        """
+        try:
+            payload = {
+                'exp': datetime.datetime.utcnow() + datetime.timedelta(days=0, minutes=60, seconds=0),
+                'iat': datetime.datetime.utcnow(),
+                'sub': user_id
+            }
+            return jwt.encode(
+                payload,
+                # TODO
+                'SECRET_KEY',
```

Comment:
yeah, can just store this in .env and then in config - similar to before

Reply:
that's what I was thinking too, but something broke when I did that. I'm sure it was a typo, but that's the plan so that these aren't hardcoded in